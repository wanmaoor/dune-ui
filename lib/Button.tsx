import React from "react"

const Button = () => <button>按钮123</button>
export default Button
