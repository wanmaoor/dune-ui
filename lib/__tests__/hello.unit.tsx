// test('hello world', ()=>{
//   expect(1).toEqual(2)
// })
import React from 'react';
import renderer from 'react-test-renderer'
import Button from "../Button";
function add(a: number, b: number){ return a + b}
describe('我的测试', ()=>{
  it('should 1 + 2 = 3', function () {
    expect(add(1,2)).toEqual(3)
  });
})

describe('button test', ()=>{
  it('should be a div tag', function () {
    const json = renderer.create(<Button />).toJSON()
    expect(json).toMatchSnapshot()
  });
})
