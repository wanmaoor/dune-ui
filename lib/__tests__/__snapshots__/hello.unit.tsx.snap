// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button test should be a div tag 1`] = `
<button>
  按钮123
</button>
`;
